### ISSUE TITLE
Reduced reciprocal kicking

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5943]
**Preferred term label:** Reduced reciprocal kicking

**Synonyms**

**Definition (free text, please give PubMed ID)** Less kicking in one leg after the other leg kicked.

**Parent term (use hpo.jax.org/app)** Abnormality of movement HP:0100022

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
https://academic.oup.com/ptj/article/84/2/159/2805351

**Comment by pnrobinson** [comment]
I have not been able to find a proper definition of what is considered reduced reciprocal kicking in PubMed. We need a definition that comprises what it is and how it is elicited, what ages it is observed at and how to determine if it is clearly abnormal. I will try again later but if you have some material on this please comment.

**Comment by pnrobinson** [comment]
I cannot find a definition of this item, and I am not convinced it is a clear abnormal finding. I will close this issue for now. If I am wrong, please provide a PMID reference with a  definition of how to measure the finding and to determine whether it is abnormal and what age range.
